frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmListModel.java

```diff
@@ -1339,31 +1339,37 @@ private void onNewTemplate() {
     private void postNameUniqueCheck() {
         UnitVmModel model = (UnitVmModel) getWindow();
         VM vm = getSelectedItem();
+        // populating VMInit
+        AsyncQuery<VM> getVmInitQuery = new AsyncQuery<>(completeVm -> {
+            VM newVm = buildVmOnNewTemplate(model, completeVm);
```

Comment:
Do I understand it correctly that the `completeVm` now contains the `vnInit`? Where is it then copied to the vm that is send to backend?

Reply:
> Do I understand it correctly that the `completeVm` now contains the `vnInit`? 

yes

> Where is it then copied to the vm that is send to backend?

in `buildVmOnNewTemplate`, see [below](https://github.com/oVirt/ovirt-engine/pull/760/files#diff-dd6dcf22431f725cccf893c4b87b0943d0e88e2b0923235a6d99e9fed2921814R1372)

Reply:
Sorry, I missed that.